oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -430,6 +435,7 @@ public IdToken idTokenWithAudience(String targetAudience, List<IdTokenProvider.O
       throw new IOException(METADATA_RESPONSE_EMPTY_CONTENT_ERROR_MESSAGE);
     }
     String rawToken = response.parseAsString();
+    // TODO: log payload here
```

Comment:
FYI in case this was forgotten

Reply:
done in [eaca20b](https://github.com/googleapis/google-auth-library-java/pull/1586/commits/eaca20b1718184eb0f44deed640ef9f71937b977)